system/services/admin/AuditService.cfc

```diff
@@ -32,6 +42,49 @@ component output="false" singleton=true {
 		} );
 	}
 
+	public query function getAuditLog(
+		  numeric startRow = 1
+		, numeric maxRows  = 0
+		) output=false {
```

Comment:
The closing `)` should align with the beginning of the function definition. Also, `output=false` is not required for script based components (even though lots of old code has it - we need to start removing them when we find them"). So:

```
public query function getAuditLog(
	  numeric startRow = 1
	, numeric maxRows  = 0
) {
```